docs/js.md

```diff
@@ -32,37 +32,6 @@ setInterval({ () =>
 }, 100)
 ```
 
-<p></p>
```

Comment:
mhtml is not published for SJS 1 and I didn't have it in me to convert it and wait for it to get published...

If an example like this is important, I could use Laminar or Outwatch as they're both pure scala js and published for SJS 1?

Reply:
It's fine to remove the mthml example, we can move forward with this PR without an example. I think it would be nice to have a real-world example with a widely used library at some point though